src/planning/include/planning/outliers.hpp

```diff
@@ -0,0 +1,46 @@
+#ifndef SRC_PLANNING_INCLUDE_PLANNING_OUTLIERS_HPP_
+#define SRC_PLANNING_INCLUDE_PLANNING_OUTLIERS_HPP_
+
+#include "common_lib/competition_logic/color.hpp"
+#include "common_lib/structures/cone.hpp"
+#include "utils/outliers_config.hpp"
+#include "utils/splines.hpp"
+
+using Cone = common_lib::structures::Cone;
+using Color = common_lib::competition_logic::Color;
+
+/**
+ * @brief Class to remove outliers
```

Comment:
I'd change to "deal with outliers", since specially in this method, we don't actually remove any